doc/components.md

```diff
@@ -185,57 +185,76 @@ For a working example, check this [source file](/doc/examples/flare/lib/main_com
 
 ## ParallaxComponent
 
-This Component can be used to render pretty backgrounds by drawing several transparent images on top of each other, each dislocated by a tiny amount.
+This Component can be used to render pretty backgrounds by drawing several transparent images on top of each other,
+where each image is moving with a different velocity.
 
 The rationale is that when you look at the horizon and moving, closer objects seem to move faster than distant ones.
 
 This component simulates this effect, making a more realistic background with a feeling of depth.
 
-Create it like this:
+The simplest `ParallaxComponent` is created like this:
 
 ```dart
-  final images = [
-    ParallaxImage('mountains.jpg'),
-    ParallaxImage('forest.jpg'),
-    ParallaxImage('city.jpg'),
-  ];
-  this.bg = ParallaxComponent(images);
+  @override
+  Future<void> onLoad() async {
+    final parallaxComponent = await ParallaxComponent.load(
```

Comment:
I feel that this will encourage people to use the global cache instead of the local game cache.

I think we could do two things to make that point clearer, let me list them here and we can discuss.

 - We could add the images parameter here
 - (My preferred) Since the Parallax isn't usual as Sprite/SpriteAnimation are I don't think we should have a `loadParallax` directly on the game class. But we could do that method as an extension on the parallax file, and here on the docs we could use that method instead.
 - (I am ok with this too) A mix of both, add the images parameter on the doc, and implement the extension method.

Reply:
Is it maybe better to have those extension methods on `ParallaxComponent`, since that would be the one that would most commonly be imported when used with the game?

Reply:
Hmm, good question, I think it should be on Parallax, for the cases where the component system is not used...maybe we could have both? `loadParallax` and `loadParallaxComponent` so we have all covered?

Reply:
Alright, like this? (Updated)

Reply:
excellent